cmake/Modules/FindSYCLToolkit.cmake

```diff
@@ -40,6 +41,24 @@ find_file(
 # Due to the unrecognized compilation option `-fsycl` in other compiler.
 list(APPEND SYCL_INCLUDE_DIR ${SYCL_INCLUDE_SYCL_DIR})
 
+# Find include/sycl/version.hpp to fetch sycl compiler version
+if(EXISTS ${SYCL_INCLUDE_SYCL_DIR}/version.hpp)
```

Comment:
better to use find_file, to align with above lines, for example line #34

Reply:
Done.